src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import Bookreducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  categories: categoriesReducer,
+  book: Bookreducer,
+});
+
+const store = configureStore({ reducer: rootReducer });
+export default store;
```

Comment:
- [ ] [OPTIONAL] I think it is better to use some handy Redux middleware, like a redux logger.

it will help you a lot in the development process. To install it,you need to run:
```react
npm i --save redux-logger
```

Reply:
@hiromataba  I have no clue about that and even what it helps for.
Hopefully, once we get introduced to it I will be using it.